requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
 -r base.txt
 
- black==19.3b0
\ No newline at end of file
+black==19.10b0
+docker-compose==1.24.1
```

Comment:
I think that who have a `Docker` already have a `docker-compose` installed too.
maybe, this create a hard flow, first run virtualenv, install dependencies and finally run `docker-compose up`

Reply:
Since it's on dev requirements only (it doesn't affect production build), I think it's better, then everything you need to run the project is in the required dependencies. What do you think?

Reply:
Perfect, batteries included 🔋.
my point is, when we running locally, maybe already have the docker-compose installed.
But, it is not block or break something when we already have docker-compose in machine.
For me, it is ok.